release.sh

```diff
@@ -37,11 +37,12 @@ echo
 make
 
 mkdir -p dist
-cd dist
 
 echo "zipping files..."
 
-zip "$TARGET_ZIP_NAME" -r ../css/
+zip "dist/$TARGET_ZIP_NAME" -r css/
+
+cd dist
```

Comment:
Instead of doing `cd dist` here, you can just add `dist/` to the `--data-binary` path.